extensions/sftp/tests/FetchSFTPTests.cpp

```diff
@@ -142,7 +142,7 @@ class FetchSFTPTestsFixture {
     std::fstream file;
     std::stringstream ss;
     ss << src_dir << "/vfs/" << relative_path;
-    utils::file::FileUtils::create_dir(utils::file::FileUtils::get_parent_path(ss.str())); // TODO
+    utils::file::FileUtils::create_dir(utils::file::FileUtils::get_parent_path(ss.str()));  // TODO
```

Comment:
the comment should be deleted

Reply:
I would refrain from deleting `TODO`-s, someone might know what they stand for. I could not trace it back what it could mean, it was added along with the function by Daniel Bakai.